compiler/src/iree/compiler/Codegen/LLVMGPU/test/vector_distribution_pipeline_rocm.mlir

```diff
@@ -55,3 +55,44 @@ hal.executable.variant @rocm target(<"rocm", "rocm-hsaco-fb", {
 // CHECK-COUNT-16:     amdgpu.mfma {{.*}} {blocks = 1 : i32, k = 16 : i32, m = 16 : i32, n = 16 : i32} blgp =  none : vector<4xf16>, vector<4xf16>, vector<4xf32>
 //          CHECK:     scf.yield %{{.+}} : vector<2x4x1x1x1x4xf32>
 //  CHECK-COUNT-8:   vector.transfer_write {{.+}} {in_bounds = [true, true]} : vector<4x1xf32>, memref<256x256xf32, #hal.descriptor_type<storage_buffer>>
+
+// -----
+
+#pipeline_layout = #hal.pipeline.layout<push_constants = 0, sets = [
```

Comment:
Can we move this to `LLVMGPU/test/ROCDL/config_matrix_core.mlir` or something? Given this is just test configuration. Start to organize things better for the new pipeline and once we move over to the new ROCDL path it's easier / less tests to migrate. :)

Reply:
oh actually I need to make this a config test, I just moved the file

Reply:
I also moved the pipeline test, let me know if that isn't what you had in mind.